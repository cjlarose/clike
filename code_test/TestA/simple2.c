
void printchar(char);

void main()
{
  char x;

  x = 65;  // the letter 'A'
  printchar(x);
}
  
