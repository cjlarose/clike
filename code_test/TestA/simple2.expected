Loaded: /usr/local/share/spim/exceptions.s
A